docs/serverless/03-03-registries.md

```diff
@@ -0,0 +1,75 @@
+
+Serverless supports two ways of connecting to an external registry:
+
+- [You can set up an external registry before installation](#tutorials-set-an-external-docker-registry).
+
+  In this scenario, you can use Kyma overrides to change the default values supplied by the installation mechanism.
+
+- [You can switch to an external registry in the runtime](#tutorials-switch-to-an-external-docker-registry).
+
+  In this scenario, you can change the registry on the fly, with Kyma already installed on your cluster. This option gives you way more flexibility and control over the choice of an external registry for your Functions.
+
+## Switching registries in the runtime
+
+When you install Kyma with the default internal registry, Helm creates the `serverless-registry-config-default` Secret in the `kyma-system` Namespace. This Secret contains credentials used by Kubernetes to pull deployed Functions' images from the internal registry. They also allow Kaniko to push any images to the registry each time a Function is created or updated.
+
+> **NOTE:** If you [install Serverless with overrides](#tutorials-set-an-external-docker-registry), disabling the internal registry and specifying an external one to use, the `serverless-registry-config-default` Secret will contain credentials to access the specified external registry instead.
+
+Once you have Serverless up and running, you can switch to an external registry:
+
+- Per Namespace, and have even multiple external registries in a cluster, but no more than one per Namespace.
+- Cluster-wide, with this configuration overwriting any Namespace-scoped.
+
+### Namespace-scoped external registry
+
+To switch to an external registry in a given Namespace, create a Secret CR that:
+
+- Is named `serverless-registry-config`.
+- Is of type `kubernetes.io/dockerconfigjson`.
+- Has the `serverless.kyma-project.io/remote-registry: config` label.
+- Contains these keys with valid values pointing to the external registry:
+  - **username**
+  - **password**
+  - **serverAddress**
+  - **registryAddress**
+
+See this example:
+
+  ```yaml
+  apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/dockerconfigjson
+  metadata:
+   name: serverless-registry-config
+   namespace: default
+   labels:
+     serverless.kyma-project.io/remote-registry: config
+  data:
+   username: {VALUE}
+   password: {VALUE}
+   serverAddress: {VALUE}
+   registryAddress: {VALUE}
+  ```
+
+### Cluster-wide external registry
+
+If you want to switch to one external registry in the whole cluster, you must create a Secret CR in the `kyma-system` Namespace. The Secret CR must meet the same [requirements](#namespace-scoped-external-registry) as in the case of the Namespace-scoped setup, but you must also add the `serverless.kyma-project.io/config: credentials` label. This label ensures the Secret CR gets propagated to all Namespaces.
```

Comment:
```suggestion
To switch to one external registry in the whole cluster, create a Secret CR in the `kyma-system` Namespace. The Secret CR must meet the same [requirements](#namespace-scoped-external-registry) as in the case of the Namespace-scoped setup, but you must also add the `serverless.kyma-project.io/config: credentials` label. This label ensures the Secret CR gets propagated to all Namespaces.
```
That's just my preference, feel free to ignore.

Reply:
I will shorten the beginning of the sentence but would somehow like to stress the second part so will stick to `must`, if you don't mind :)